datadog/resource_datadog_dashboard.go

```diff
@@ -4593,6 +4843,162 @@ func getTimeseriesRequestSchema() map[string]*schema.Schema {
 		},
 	}
 }
+
+func buildDatadogFormula(data map[string]interface{}) datadogV1.WidgetFormula {
+	formula := datadogV1.WidgetFormula{}
+	if formulaExpression, ok := data["formula_expression"].(string); ok && len(formulaExpression) != 0 {
+		formula.Formula = formulaExpression
```

Comment:
```suggestion
		formula.SetFormula(formulaExpression)
```
Lets use the setters here

Reply:
Done 👍